src/main/java/edu/ucmo/spring_example/controller/CarController.java

```diff
@@ -43,7 +43,7 @@ public List<Car> getMake(@PathVariable String make){
     @PutMapping("/{id}")
     public Car update(@RequestBody Car carUpdate) {
         Optional<Car> optionalCar = carDao.findById(carUpdate.getId());
-        if (optionalCar.isPresent()) {
+        if (optionalCar.isPresent() && optionalCar.model == "Ford") {
```

Comment:
Why are we chekcing for Ford's that makes no sense.

Reply:
Because I want to.